### DISCUSSION TITLE
请问实现的EDGE浏览器？

### DISCUSSION THREAD

**Discussion by IZichenya** [win12-online/win12#206] [Q&A] [answered]
请问怎么实现的EDGE浏览器？

**Comment by tjy-gitnub** [comment]
我敢肯定这不是中文，因为我看不懂。

**Reply by IZichenya** [reply]
打快了，怎么实现的edge浏览器啊，

**Reply by tjy-gitnub** [reply]
iframe 嵌套网页哦~

**Reply by patrickyao0422** [reply, marked as answer]
就是一个Microsoft Edge的区块，里面嵌套了一个iframe元素，这个元素能将Html页面单独显示出来。说白了，iframe就是把访问网址的内容拷下来，粘贴到属于iframe的位置上

**Comment by IZichenya** [comment]
请问下说的EDGE区块是什么意思？



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年3月11日(周一) 晚上7:45
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [tjy-gitnub/win12] 请问实现的EDGE浏览器？ (Discussion #206)




 
就是一个Microsoft Edge的区块，里面嵌套了一个iframe元素，这个元素能将Html页面单独显示出来。说白了，iframe就是把访问网址的内容拷下来，粘贴到属于iframe的位置上

**Reply by User782Tec** [reply]
不到啊（

**Reply by patrickyao0422** [reply]
HTML知道么？里面有一个div元素，把页面分割成一个个的小块。给Edge分配一个这样的小块，你在Edge里看到的网页都包含在这小块里面
